app/src/main/java/com/kickstarter/viewmodels/CreatorDashboardHeaderHolderViewModel.java

```diff
@@ -0,0 +1,112 @@
+import com.kickstarter.libs.rx.transformers.Transformers;
+import com.kickstarter.libs.utils.NumberUtils;
+import com.kickstarter.libs.utils.ProjectUtils;
+import com.kickstarter.models.Project;
+import com.kickstarter.models.ProjectStats;
+import com.kickstarter.ui.viewholders.CreatorDashboardHeaderViewHolder;
+import com.kickstarter.viewmodels.inputs.CreatorDashboardHeaderHolderViewModelInputs;
+import com.kickstarter.viewmodels.outputs.CreatorDashboardHeaderHolderViewModelOutputs;
+
+import rx.Observable;
+import rx.subjects.BehaviorSubject;
+import rx.subjects.PublishSubject;
+
+public interface CreatorDashboardHeaderHolderViewModel {
+
+  final class ViewModel extends ActivityViewModel<CreatorDashboardHeaderViewHolder> implements
+    CreatorDashboardHeaderHolderViewModelInputs, CreatorDashboardHeaderHolderViewModelOutputs {
+
+
+    public ViewModel(final @NonNull Environment environment) {
+      super(environment);
+
+      projectAndStats
+        .map(ps -> ps.first)
+        .subscribe(this.currentProject);
+
+      projectAndStats
+        .map(ps -> ps.first)
+        .map(Project::percentageFunded)
+        .map(NumberUtils::flooredPercentage)
+        .compose(bindToLifecycle())
+        .subscribe(this.percentageFunded);
+
+      projectAndStats
+        .map(ps -> ps.first)
+        .map(Project::backersCount)
+        .map(NumberUtils::format)
+        .compose(bindToLifecycle())
+        .subscribe(projectBackersCountText);
+
+      projectAndStats
+        .map(ps -> ps.first)
+        .map(Project::name)
+        .distinctUntilChanged()
+        .compose(bindToLifecycle())
+        .subscribe(projectNameTextViewText);
+
+      projectAndStats
+        .map(ps -> ps.first)
+        .map(ProjectUtils::deadlineCountdownValue)
+        .map(NumberUtils::format)
+        .subscribe(timeRemainingText);
+
+      this.currentProject()
+        .compose(Transformers.takeWhen(projectViewClicked))
+        .map(p -> Pair.create(p, RefTag.dashboard()))
+        .subscribe(this.startProjectActivity);
+    }
+
+
+    public final CreatorDashboardHeaderHolderViewModelInputs inputs = this;
+    public final CreatorDashboardHeaderHolderViewModelOutputs outputs = this;
+
+    private final PublishSubject<Pair<Project, ProjectStats>> projectAndStats = PublishSubject.create();
+    private final PublishSubject<Void> projectViewClicked = PublishSubject.create();
+
+    private final BehaviorSubject<String> percentageFunded = BehaviorSubject.create();
+    private final BehaviorSubject<Project> currentProject = BehaviorSubject.create();
+    private final BehaviorSubject<String> projectBackersCountText = BehaviorSubject.create();
+    private final BehaviorSubject<String> projectNameTextViewText = BehaviorSubject.create();
+    private final BehaviorSubject<Pair<Project, RefTag>> startProjectActivity = BehaviorSubject.create();
+    private final BehaviorSubject<String> timeRemainingText = BehaviorSubject.create();
```

Comment:
since these outputs aren't dependent on a value coming through from the `intent()`, these outputs can actually all be `Observable`s. can chat IRL about this since it's a confusing lil gotcha about our codebase